jwst/extract_1d/soss_extract/tests/test_soss_extract.py

```diff
@@ -81,6 +81,9 @@ def test_model_image(monkeypatch_setup, imagemodel, detector_mask, detector_mode
     else:
         extract_order3 = False
 
+    for model in detector_models:
+        model.mederr = scierr
+
```

Comment:
Instead of putting this into every test, can you put it into the detector_models fixture?

Reply:
Yes, but then I would have to instantiate it with the construction of detector_models, which seemed like it would require a heavier editing hand on the tests.  I agree that it would be conceptually cleaner.  I'll have another look.